ci/kind/kind-setup.sh

```diff
@@ -321,12 +333,18 @@ function delete_vlan_subnets {
 }
 
 function delete_networks {
-  networks=$(docker network ls -f name=antrea --format '{{.Name}}')
-  networks="$(echo $networks)"
-  if [[ ! -z $networks ]]; then
-    docker network rm $networks > /dev/null 2>&1
-    echo "deleted networks $networks"
-  fi
+  local filters=("antrea" "kind")  # Define network filters to process
+  for filter in "${filters[@]}"; do
```

Comment:
instead of this, define a helper function which takes the filter as a parameter, then call the function here as needed